external/fv3fit/fv3fit/emulation/transforms/transforms.py

```diff
@@ -53,6 +53,45 @@ def backward(self, y: TensorDict) -> TensorDict:
         return out
 
 
+class ConditionallyScaledTransform(TensorTransform):
```

Comment:
Similar to the new factory class, can you make it clear in the name that this is for residuals?